app/scripts/controllers/network/network-controller.test.ts

```diff
@@ -814,8 +373,17 @@ describe('NetworkController', () => {
                 },
               },
             },
-            async ({ controller, network }) => {
-              network.mockEssentialRpcCalls();
+            async ({ controller }) => {
+              const fakeProvider = buildFakeProvider([
+                {
+                  request: {
+                    method: 'net_version',
+                  },
+                  response: SUCCESSFUL_NET_VERSION_RESPONSE,
```

Comment:
Nit: This stub seems to be included by default by `buildFakeProvider` - is this needed here?

Reply:
It is included by default, but I'm choosing to be explicit here because the test is ensuring that `networkStatus` returns "available", and that relies upon `net_version` returning a successful result. Actually, `eth_getBlockByNumber` also needs to return a successful result too. I've increased the explicitness of these tests: [02a2bcc](https://github.com/MetaMask/metamask-extension/pull/18628/commits/02a2bcc4d8ecdab9ed03864b9f59965b7bb73bdd)